src/test/java/org/cactoos/io/ZipTest.java

```diff
@@ -63,12 +63,12 @@ public void zip() throws Exception {
                 ++cnt;
                 entry = input.getNextEntry();
             }
-            MatcherAssert.assertThat(
+            new Assertion<>(
                 "Can't list files in a directory represented by a path",
                 cnt,
                 // @checkstyle MagicNumber (1 line)
                 Matchers.equalTo(4)
```

Comment:
@marceloamadeu can you change that to `new IsEqual<>(4)`?

Reply:
@victornoel Done ... please check.